src/Layout/Callout.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { FunctionComponent, ReactNode } from 'react';
+import styled from 'styled-components';
+import { VARIANT } from '../Theme';
+
+export interface CalloutProps {
+  /**
+ * Allows the user to set the background color by passing in a variant property from the VARIANT enum
+ * @default VARIANT.INFO
+ */
+  variant?: VARIANT;
+  /**
+   * The aria role of the displayed callout. A list of the different role
+   * values that should be used can be found in the w3 docs:
+   * https://www.w3.org/TR/wai-aria-1.1/#live_region_roles
+   * The default value role is 'note'.
+   */
+  role?: string;
+  children: ReactNode;
+}
+
+interface StyledCalloutProps {
+  variant: VARIANT;
+}
+
+const StyledCallout = styled.p<StyledCalloutProps>`
+  font-family: ${({ theme }): string => theme.font.callout.family};
+  font-size: ${({ theme }): string => theme.font.callout.size};
+  font-weight: ${({ theme }): string => theme.font.callout.weight};
+  color: ${({ theme }): string => theme.color.text.dark};
+  background: ${({ theme, variant }): string => theme.color.background[variant].light};
+  margin-left: ${({ theme }):string => theme.ws.large};
```

Comment:
I would recommend duplicating this for `margin-right` as well, for cases where a `Callout` might fill the entire horizontal space. Alternatively, we could define this as a single margin prop, along the lines of:

```
  margin: ${({ theme }):string => `${theme.ws.medium} ${theme.ws.large} ${theme.ws.small} ${theme.ws.large}`};
```